### PR TITLE
Add Luca's Panel from Rust for the Web Barcelona

### PR DESCRIPTION
**Pull request by IdeliaRisella**
- Add the talk to appearances
- Add the rustweb barcelona channel

To do: add thumbnail

### CODE REVIEW

File: src/channels/rustweb-barcelona.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Rust for the web"
+image: "/assets/images/talks/2024-05-30-rustweb-barcelona/logo.png"
+url: https://www.meetup.com/es-ES/bcnrust/events/300765894/
+---
+
+“Rust for the web” is an event organized by the Mainmatter team in collaboration
```
Comment (marcoow) [inline src/channels/rustweb-barcelona.md]:
```suggestion
“Rust for the web” Barcelona was an event organized by the Mainmatter team in collaboration
```

this was a one-off event

**Review by marcoow — CHANGES_REQUESTED** [review]

**Comment by IdeliaRisella** [comment]
@marcoow I tried changing the thumbnail on this PR, as well as another one, and it seems like it's not quiet right as the checks were not successful. I tried doing what Emma explained to me before. I am not sure where exactly I missed. Would you please help and explain to me what I need to pay attention to so I can work on the rest of the thumbnail, or maybe I should go to Xenia to help me learn more?

**Comment by marcoow** [comment]
There was a lint error (I fixed) and the branch was branched off of an old ref which lead to the visual changes reported by Percy. Bart or Gabor should have some time maybe to explain in more detail.

The Netlify error was random and unrelated to your changes…

**Review by marcoow — APPROVED** [review]

**Comment by marcoow** [comment]
let's make sure we get these things on the website a bit quicker after the events have occurred in the future :pray:

**Comment by IdeliaRisella** [comment]
Hi Marco,

I just want to let you know that there is another open PR made by Emma that
hasn't been merged yet. I think the same problem happened like this one.
The PR was undone because the new thumbnail was not available in July, and
I seemed to have lost track of this task. But I'm planning to get this done
this week, as well as changing all the thumbnails. I'll contact Bart or
Gabor for further explanation, thank you again.

Best regards,
Idelia Risella

**Pull request merged**